src/guides/v2.4/graphql/mutations/delete-requisition-list.md

```diff
@@ -43,7 +51,11 @@ mutation {
 {
   "data": {
     "deleteRequisitionList": {
-      "result": true
+      "status": true,
+      "requisition_lists": {
```

Comment:
i think this needs to return pagination then status

type RequisitionLists @doc(description: "Defines customer requisition lists")  {
    items: [RequisitionList] @doc(description: "An array of requisition lists")
    page_info: SearchResultPageInfo @doc(description: "Contains pagination metadata")
    total_count: Int @doc(description: "The number of returned requisition lists")
}